### PR TITLE
17.10

### PR DESCRIPTION
**Pull request by deadlight**
Just making a demo

### CODE REVIEW

File: templates/desktop/1710.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
+{% extends "desktop/base_desktop.html" %}
+
+{% block title %}Ubuntu 17.10{% endblock %}
+{% block meta_description %}Fast, secure and stylishly simple, the Ubuntu operating system is used by 50 million people worldwide every day.{% endblock %}
+
+{% block extra_body_class %}1710{% endblock %}
+
+{% block content %}
+
+<section class="p-strip is-bordered">
```
Comment (anthonydillon) [inline templates/desktop/1710.html]: We normally make this `is-deep` so the titles line up across the pages.

File: templates/server/index.html [hunk 2]
```diff
@@ -257,8 +258,8 @@ <h2 id="built-for-big-data">Built for big data and cloud</h2>
...
+      <p>Ready to boost efficiencies and reduce costs? Virtualise your servers with Ubuntu Server, KVM and LXD. When you use a secure, lean version of Ubuntu as a guest operating system for your application, you can create virtual machines and machine containers in seconds. KVM, LXD, Xen, VMware, Vagrant, VirtualBox, and Docker are all first class experiences with Ubuntu Server. Ubuntu Server now supports KVM on ARM and IBM POWER8 architectures.</p>
+      <p>Canonical believes in choice - that's why we support an open ecosystem of operating system intercompatibility. By providing choice, users of Ubuntu for both servers and clouds are empowered to succeed with any workload. We encourage others to do the same and offer support for all (currently supported) versions of Ubuntu Server running on other virtualisation platforms, including KVM on other commercially available Linux distributions, VMware vSphere, and Microsoft Hyper&#8209;V.</p>
```
Comment (anthonydillon) [inline templates/server/index.html]: The dash and ' could do with encoding.

**Comment by anthonydillon** [comment]
The demo doesn't seem to have vanilla 1.6.0? Has it been reverted?

**Comment by deadlight** [comment]
Ok, we've merged in everything we're going to go live with.
Take a look!

File: templates/server/index.html (hunk 2 above)
Comment (deadlight) [reply templates/server/index.html]: I'll fix these.

**Comment by anthonydillon** [comment]
- QA issues:
On the /download/desktop/contribute we need to set `.contribute__option-value` to `margin-top: 0;`

**Comment by frankiegranato** [comment]
Some weirdness with external link icons - see /download/desktop
Wrong time on the takeover screenshot

**Comment by deadlight** [comment]
@frankiegranato added a fix for the external icons

**Pull request merged**